integration-tests/trailing-slash/test/index.test.ts

```diff
@@ -24,6 +24,25 @@ const runTests = (mode?: string) => {
       5000 * 60 * 2,
     )
   })
+
+  describe("get query data", () => {
+    it(
+      "should work",
+      async () => {
+        const browser = await webdriver(appPort, "/page-with-get-query-data")
+
+        browser.waitForElementByCss("#button", 0)
+        await browser.elementByCss("#button").click()
+
+        browser.waitForElementByCss("#new-data", 0)
+        const newText = await browser.elementByCss("#new-data").text()
+        expect(newText).toMatch(/basic-result/)
+
+        if (browser) await browser.close()
+      },
+      5000 * 60 * 2,
+    )
+  })
```

Comment:
This is awesome, but is it possible to have this test in `integration-tests/rpc` or in a new folder? Asking because tests in this folder should only be related to trailing slash.

Reply:
Sure. I’ll have a look at this tomorrow

Reply:
Hi @beerose, I need help with this. I'm trying to add this into the `rpc` integration test but that means pulling in `@blitzjs/rpc` and also `@blitzjs/auth` (because of https://github.com/blitz-js/blitz/issues/3663). Pulling in `@blitzjs/auth` means I have to add prisma. Unfortunately, serverless mode which is part of the tests, doesn't work with this. What should I do here?

Reply:
Ah, I see.

Hmm, there's only one test in the `trailing-slash`, and to be honest, I'm not even sure if it's testing the trailing slash thing correctly (https://nextjs.org/docs/api-reference/next.config.js/trailing-slash) 🤷‍♀️

I see two options:
- Renaming this directory `trailing-slash` -> `react-query-utils`. But then we lose the trailing-slash context.
- Copying the whole trailing slash directory into `integration-tests/react-query-utils` and putting this test there (over time, we should have more tests there).

Reply:
Thanks. I've duplicated the trailing slash directory into a new `react-query-utils` and the tests are working locally now.